routes/chat.js

```diff
@@ -84,6 +86,13 @@ router.post('/chat', async function (req, res) {
       chat: chatId,
     });
 
+    const userSocket = [io.sockets.adapter.rooms.get(req.userId)];
+    const otherSocket = [io.sockets.adapter.rooms.get(otherId)];
+    serverSocket.joinRooms(chatId.toString(), otherSocket);
```

Comment:
Instead of passing one user at a time, it would be more beneficial to give an array of users to be then used to create groups.